### DISCUSSION TITLE
0.2.8 Release?

### DISCUSSION THREAD

**Discussion by LuvForAirplanes** [odoo/odoo-ls#162] [General]
@fda-odoo when will the next version be available? I would like to try it on my Windows PC. The extension loads some things, but it completely ignores my custom code.
It's been a very hair raising experience.

**Comment by fda-odoo** [comment]
Hello @LuvForAirplanes ,

The next version should come this month, but we don't give date as we try to release versions when new features are considered stable enough to be public.
However, the next version won't change a lot of things in the interface, so if you have issues currently, I would be happy to know more about it, so we can try to fix it. There is as many setup that there is developers, so maybe we forgot to handle some configurations, so if you can give me more information, it would be really nice !

- It may seem a dumb question, but did you read the tutorial that show up when you install the extension to configure it? you can open it again at any time by pressing ctrl+R then typing "odoo welcome page"
- Is your custom code in a module? Or are you using the Odoo framework from outside without using the module system? Actually, the language server is only able to parse and give features for Odoo and all the addon paths.
- Did you add your addon path with your current module in the configuration?

We are ready to hear your feedback or your issues and to help you getting a nice experience with the tool! As you saw it, the tool is still in development, and any feedback is welcome!

**Comment by LuvForAirplanes** [comment]
Good morning, @fda-odoo,
I outlined my setup below, which should hopefully give you more clues about what could be going wrong:
- System is running Linux Mint 22
- The main Odoo 17 installation is in /opt/odoo (git clone from github.com/odoo/odoo) single source folder being so we can share installations between custom projects
- The custom code is located in:
  - /opt/crop-doc/addons
  - /opt/crop-doc/themes
  - /opt/crop-doc/enterprise/modules
- The venv is here: /opt/crop-doc/venv

And here's my VS Code configuration:
```
        "0": {
            "id": 0,
            "name": "Crop-Doc",
            "odooPath": "/opt/odoo",
            "rawOdooPath": "/opt/odoo",
            "addons": [
                "/opt/crop-doc/addons",
                "/opt/crop-doc/odoo-enterprise/modules",
                "/opt/crop-doc/themes"
            ],
            "pythonPath": "/opt/crop-doc/venv/bin/python3"
        }
```

And here is the log I'm getting when I try to activate the profile:
[log.log](https://github.com/user-attachments/files/18182951/log.log)

Let me know what you find.

**Reply by LuvForAirplanes** [reply]
My first reaction was: Well what is going on with the `Failed to read files` errors? But I can nano into them just fine, and they are owned by my home user. If I run whoami from the terminal that VS Code creates, it comes back with micah, which is what _everything_ should be owned by.

**Comment by fda-odoo** [comment]
Hello,

Thank you for giving us more context about your issue. I can't find any issue in your configuration or meaningful information in logs :'(
You can safely ignore "File not found" errors, it will only concern external sources and won't prevent the tool for working for now. We have these errors in logs too on our side, without crash.
Your server seems to crash when the server close, so it is not what is preventing you from using it.

At this point, I would suggest you to wait the 0.2.8 as it will be out this week, and see if you still have the issue. If yes or if you want to continue debugging your 0.2.7, can you try to lower the log level before restarting the extension?
You can find the option in Preferences/settings/type "Odoo"/Server Log Level => debug
and Odoo Server > Trace: Verbose.
After launching the extension, logs should be more verbose, and you'ill find communication channel with vscode in your vscode output window too (View/Output/change the dropdown to "Odoo"). Do not hesitate to communicate them to me if it doesn't solve your issue.

I remain at your disposal should you have any question or remark

**Comment by LuvForAirplanes** [comment]
I really appreciate the work that is going into this. Keep it up @fda-odoo the latest version (0.4.1 is working fantastically in VS Code)
